Constance/base.py

```diff
@@ -0,0 +1,7 @@
+class BaseConst:
```

Comment:
Typo: Constance is a name ;) 
constants is a directory. Also we're naming python packages using lowercase